87.md

```diff
@@ -0,0 +1,129 @@
+  "pubkey": <recommender-user-pubkey>,
+  "tags": [
+    ["u", <recommended-cashu-mint-url>, "cashu"],
+    ["u", <recommended-fedimint-invite-code>, "fedimint"],
+    ["u", <recommended-cashu-mint-url>, "cashu"],
+    ["a", "38173:mint1-pubkey:<d-identifier>", "wss://relay1", "cashu"],
+    ["a", "38173:mint2-pubkey:<d-identifier>", "wss://relay2", "fedimint"]
+  ]
+}
+```
+
+`u` tags are used to point directly to the ecash mint.
+The first value of the tag is the URL or invite code of the ecash mint, and the second value is the type of ecash mint (currently `cashu` or `fedimint`).
+Multiple `u` tags can appear on the same `kind:18173`.
+
+`a` tags are used to point to the `kind:38173` event of the ecash mint.
+The first value of the tag is the `kind:38173` event identifier, the second value of the tag is a relay hint, and the third value of the tag is the type of ecash mint (currently `cashu` or `fedimint`).
+
+## Ecash Mint Information
+
+Cashu mints SHOULD publish `kind:38173` events to announce their capabilities and how to connect to them.
+
+For cashu mints, the `u` tag SHOULD be the URL to the cashu mint and it should list the nuts that the cashu mint supports.
+
+```json
+{
+  "kind": 38173,
+  "pubkey": "<application-pubkey>",
+  "content": "<optional-kind:0-style-metadata>",
+  "tags": [
+    ["d", <random-id>],
+    ["u", "https://cashu.example.com", "cashu"],
+    ["nuts", "1,2,3,4,5,6,7"]
+  ]
+}
+```
+
+Fedimints SHOULD publish `kind:38173` events to announce their capabilities and how to connect to them.
+
+For fedimints, it should list all known fedimint invite codes in `u` tags and it should list the modules it supports.
+
+```json
+{
+  "kind": 38173,
+  "pubkey": "<application-pubkey>",
+  "content": "<optional-kind:0-style-metadata>",
+  "tags": [
+    ["d", <random-id>],
+    ["u", "fed11abc..", "fedimint"],
+    ["u", "fed11xyz..", "fedimint"],
+    ["modules", "lightning,wallet,mint"]
+  ]
+}
+```
+
+* `content` is an optional `metadata`-like stringified JSON object, as described in NIP-01. This content is useful when the pubkey creating the `kind:38173` is not an application. If `content` is empty, the `kind:0` of the pubkey should be used to display application information (e.g. name, picture, web, LUD16, etc.)
+
+## User flow
+A user A who uses a non-`kind:1`-centric nostr app could choose to announce/recommend a certain kind-handler application.
+
+When user B sees an unknown event kind, e.g. in a social-media centric nostr client, the client would allow user B to interact with the unknown-kind event (e.g. tapping on it).
+
+The client MIGHT query for the user's and the user's follows handler.
+
+## Example
+
+### User A recommends a cashu mint
+User A might be a user of a cashu mint. Using a client, user A publishes an event recommending the cashu mint they use.
+
+```json
+{
+  "kind": 18173,
+  "tags": [
+    ["u", "fed11abc..", "fedimint"],
+    ["u", "https://cashu.example.com", "cashu"]
+  ],
+  ...
+}
+```
```

Comment:
Same thing here. The user recommends a federation mint. Not a federation invite code.